cogs/tables.py

```diff
@@ -25,8 +25,11 @@ async def leagueTable(self, ctx,
         # Check if (w)hite or (l)ight, else use dark mode
         light_mode = background.startswith('l') or background.startswith('w')
 
-        bg_colour = (255, 255, 255) if light_mode else (47, 49, 54)
-        font_colour = (0, 0, 0) if light_mode else (255, 255, 255)
+        if (light_mode):
```

Comment:
Not sure why the brackets here. Anyways in the future we will introduce Black or similar so we all write similar styles :)